sljit_src/sljitNativeMIPS_32.c

```diff
@@ -26,6 +26,23 @@
 
 /* mips 32-bit arch dependent functions. */
 
+static sljit_s32 emit_copysign(struct sljit_compiler *compiler, sljit_s32 op,
+		sljit_sw src1, sljit_sw src2, sljit_sw dst)
+{
+	int is_32 = (op & SLJIT_32);
+	sljit_ins fr_pair = is_32 ? 0 : (1 << 11);
+
+	if (!is_32 && src1 != dst)
+		FAIL_IF(push_inst(compiler, MOV_fmt(FMT_D) | FS(src1) | FD(dst), DR(TMP_REG1)));
```

Comment:
Move this down before the xor, and change draft to ready for review. Is it better to move the whole 64 bit instead of the upper 32? I have no idea which one is better.

Reply:
> Is it better to move the whole 64 bit instead of the upper 32?

I don't think that is supported by the instruction as described in the documentation:

```
The operand must be a value in format fmt; if it is not, the result is UNPREDICTABLE and the value of the operand
FPR becomes UNPREDICTABLE.
```

Reply:
The format would be single precision, and +1 is added or src and dst. Anyway, if you say this is good, I am ok with it. I don't know if it causes dependency issues or the opposite.